sources/packages/web/src/composables/useFormioUtils.ts

```diff
@@ -164,6 +164,21 @@ export function useFormioUtils() {
     return associatedFiles;
   };
 
+  // Get all original file names from all file components.
+  const getOriginalFileNames = (form: any): string[] => {
+    const fileComponents = getComponentsOfType(form, "file");
+    const originalFileNames: string[] = [];
+    fileComponents.forEach((fileComponent) => {
+      const fileComponentValue = fileComponent.getValue();
+      if (fileComponentValue) {
+        fileComponentValue.forEach((file: any) => {
+          originalFileNames.push(file.originalName);
+        });
+      }
+    });
+    return originalFileNames;
+  };
```

Comment:
question about the approach of obtaining the file friendly name. We can obtain the same at backend right? why do we need UI to send them ?

Reply:
Ok, I can change that.